src/components/ExportCSV/ExportCSVForm/ExportCSVFormMobile.jsx

```diff
@@ -550,16 +600,41 @@ const ExportCSVForm = ({ formID }) => {
             </DrawerBody>
           </DrawerContent>
         </Drawer>
-      </form>
 
-      <div className={styles['buttons-container']}>
-        <Button border="1px" borderColor="#CBD5E0" bg="white">
-          Cancel
-        </Button>
-        <Button textColor="white" bg="#345E80">
-          Preview CSV
-        </Button>
-      </div>
+        <Drawer onToggle={customCountryInputToggle} isOpen={customCountryInputOpen} size="full">
+          <DrawerContent>
+            <ChevronLeftIcon
+              className={styles['back-button']}
+              boxSize={7}
+              onClick={() => (isValid ? customCountryInputToggle() : console.log(isValid))}
+            />
+            <DrawerHeader className={styles['additional-input-header']}>Country</DrawerHeader>
+            <DrawerBody>
+              <FormControl isInvalid={errors?.sortBy}>
+                <FormLabel htmlFor="country">Custom</FormLabel>
+                {watchAllFields.countryOption === 'country-custom' && (
+                  <Controller
+                    control={control}
+                    name="country"
+                    // eslint-disable-next-line no-unused-vars
+                    render={({ field: { onChange, value, ref } }) => (
+                      <ReactSelect options={countryOptions} value={value} onChange={onChange} />
+                    )}
+                  />
+                )}
+              </FormControl>
+            </DrawerBody>
+          </DrawerContent>
+        </Drawer>
+        <div className={styles['buttons-container']}>
+          <Button border="1px" borderColor="#CBD5E0" bg="white" onClick={() => navigate('/')}>
+            Cancel
+          </Button>
+          <Button textColor="white" bg="#345E80" type="submit">
+            Preview CSV
+          </Button>
+        </div>
```

Comment:
Could we make these buttons sticky and at the bottom of the page? When the user opens one of the filtering options, the dropdown shouldn't push the buttons down.